pkg/Microsoft.NETCore.Platforms/Microsoft.NETCore.Platforms.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   
   <PropertyGroup>
-    <PackageVersion>1.0.2</PackageVersion>
+    <PackageVersion>1.2.0</PackageVersion>
```

Comment:
I made a similar suggestion to @chcosta, he said he had trouble with it.  I can see if I can make that change.